src/tests/Parser/Brewmaster/Fixtures/SimpleFight.js

```diff
@@ -0,0 +1,60 @@
+import SPELLS from 'common/SPELLS';
+
+export const FIGHT_END = 11000;
+export const TOTAL_STAGGERED = 240;
+
+const thisPlayer = 1;
+const enemy = 2;
+
+export function processEvents(events, module, fightEndTime) {
```

Comment:
I have moved it out into its own file because i need to have more than one fight for edge cases, i'll maybe look at moving it later though.